python/ambassador/ir/irauth.py

```diff
@@ -126,6 +127,11 @@ def _load_auth(self, module: Resource, ir: 'IR'):
             if add_linkerd_headers is None:
                 self["add_linkerd_headers"] = ir.ambassador_module.get('add_linkerd_headers', False)
 
+        if module.get('circuit_breakers', None):
+            self['circuit_breakers'] = module.get('circuit_breakers')
+        else:
+            self['circuit_breakers'] = ir.ambassador_module.circuit_breakers
```

Comment:
Feels like this needs to use `ir.ambassador.module('circuit_breakers', None)` to avoid a crash if the `ambassador` ``Module` has no `circuit_breakers` key... ?

Reply:
Huh. Yeah, it does seem like that.

But this works, and I don't have an `ambassador` `Module`, which I think means it's safe?

Reply:
Looking over the code in `irambassador.py`, I think that this is an accident of the current `irambassador` implementation. I'm rebasing this onto `master`, so I'll tweak this as well.